src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const array = [];
+const spans = document.querySelectorAll('.population');
+
+for (const population of spans) {
+  array.push(population.innerText);
+}
```

Comment:
Let's use array iteration methods

Reply:
Hint: NodeList that returned from querySelectorAll can be easily converted to array spread operator